dependency-graph/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Dependency Review",
+    "creator": "GitHub",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Dependency Graph",
```

Comment:
I am not too sure if this starter workflow would show up in `Security` section. Not sure but I think that is hard coupled today with `Code scanning` category.
@anuragc617 @Phantsure can you please confirm. If so, can we quickly fix this in the UI to allow this to show up.

Reply:
Got a reply below, moved the files to `code-scanning/` top-level dir tree. Let me know if there's more related changes to enable this (temporary) home for security gallery entries 👍